ansible/wireguard_sn10.yaml

```
@@ -128,7 +128,7 @@ wireguard_configs:
     INTERFACE_ADDRESS: "10.70.247.16/31"
 
     # For Jimmy laptop
-  - NAME: jimmylaptop
+  - NAME: jimmy
```

```suggestion
  - NAME: jimmylaptop
```
Changing interface names is not currently supported. You don't have to use this value, not need to change it.